source/how-tos/boot-zephyr.rst

```diff
@@ -0,0 +1,101 @@
+.. _boot-zephyr:
+
+Booting Zephyr
+-----------------
+
+We may follow below steps to setup Zephyr with |SPN| on |UP2| maker board and QEMU.
+
+|UP2| Board
+^^^^^^^^^^^^^^^^^
+
+1. Follow the '|Getting Started Guide|' to setup **Zephyr** build/dev environment
+
+.. |Getting Started Guide| raw:: html
+
+   <a href="https://docs.zephyrproject.org/latest/getting_started/getting_started.html" target="_blank">Getting Started Guide</a>
+
+2. Build an application uses synchronization sample application:
+
+  #. setup zephyr environment::
+  
+        source zephyr-env.sh
+        or source zephyr-env.cmd
```

Comment:
or zephyr-env.cmd

Remove 'source' for windows environment. Sorry for missing this in earlier comment.